.github/workflows/deploy-demo.yaml

```diff
@@ -1,11 +1,7 @@
 name: Deploy Demo
 
 on:
-  push:
-    branches: [ main ]
-    paths:
-      - 'demo/**'
-      - '.github/workflows/deploy-demo.yaml'
+  workflow_call:
```

Comment:
Deploying demo every push now so the version on the site matches this published version